pkg/plugin/restore_pvc_action_plugin.go

```diff
@@ -139,6 +139,22 @@ func (p *NewPVCRestoreItemAction) Execute(input *velero.RestoreItemActionExecute
 		return nil, errors.WithStack(err)
 	}
 
+	// Retrieve storage class mapping information and update pvc StorageClassName with new name
+	storageClassMapping := make(map[string]string)
+	p.Log.Info("Retrieving storage class mapping information from configMap")
+	err = pluginItem.RetrieveStorageClassMapping(restConfig, veleroNs, storageClassMapping, p.Log)
```

Comment:
It's better to specify storageClassMapping as an output parameter, not an input parameter as it is empty as input.